spec/webmock/user_spec.rb

```diff
@@ -95,5 +95,42 @@
       # Assert the response and test your code's behavior
       expect(response['success']).to eq('false')
     end
+
+    ###################################################################################################################
+    # POST USER TEST
+    # #################################################################################################################
+    it 'Creates a new user' do
+      body = {
+        'first_name' => 'Jack',
+        'last_name' => 'Smith',
+        'email' => 'jack.smith@email.com'
+      }
+
+      stub_request(:post, "#{host}/api/user")
+        .to_return(status: 200, body: body.to_json)
+
+      response_json = client.create_user(first_name: 'Jack', last_name: 'Smith', email: 'jack.smith@email.com',
+                                         user_group_memberships: nil, login_attributes: nil)
+      new_user = JSON.parse(response_json)
+
+      # Assert the response and test your code's behavior
+      expect(new_user['email']).to eq('jack.smith@email.com')
+    end
+    # tries to create a user with an existing email
+    it 'Failed to create user with same email' do
+      body = {
+        'email' => 'email address already in use'
+      }
+
+      stub_request(:post, "#{host}/api/user")
```

Comment:
Based on a real response, the status should be 400 BadRequest.  Best way I found to test this is has shown
https://github.com/mrgutie8/metabase-ruby-rest-api/blob/e3580d8afdfd63e32c560cbc4d6fc7fe0eabccc4/spec/webmock/user_spec.rb#L53C1-L63C10